opta/commands/secret.py

```diff
@@ -84,12 +81,10 @@ def update(secret: str, value: str, env: Optional[str], config: str) -> None:
     _raise_if_no_k8s_cluster_exists(layer)
 
     configure_kubectl(layer)
+    create_namespace(layer.name)
+    create_manual_secrets(layer.name)
```

Comment:
I think it would be cleaner to call `create_manual_secrets` from `update_manual_secrets` itself

Reply:
done